src/components/MenuPanel/MenuPanelFooter.js

```diff
@@ -2,19 +2,61 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import { Button, IconSettings, Text, theme } from '@aragon/ui'
+import { EthereumAddressType } from '../../prop-types'
+import LocalIdentityBadge from '../../components/IdentityBadge/LocalIdentityBadge'
+
+const MenuPanelFooter = ({
+  account,
+  connected,
+  onOpenPreferences,
+  onRequestEnable,
+}) => {
+  return (
+    <div css="margin: 15px 20px">
+      <ConnectionBullet connected={connected} />
+      <Text size="xsmall">
+        {connected ? 'Connected to the network' : 'Not connected'}
+      </Text>
+      <ConnectedAccount account={account} onRequestEnable={onRequestEnable} />
+      <PreferencesButton onClick={onOpenPreferences} />
+    </div>
+  )
+}
 
-const MenuPanelFooter = ({ connected, onOpenPreferences }) => (
-  <div css="margin: 15px 20px">
-    <ConnectionBullet connected={connected} />
-    <Text size="xsmall">
-      {connected ? 'Connected to the network' : 'Not connected'}
-    </Text>
-    <PreferencesButton onClick={onOpenPreferences} />
-  </div>
-)
 MenuPanelFooter.propTypes = {
+  account: EthereumAddressType,
   connected: PropTypes.bool,
   onOpenPreferences: PropTypes.func.isRequired,
+  onRequestEnable: PropTypes.func.isRequired,
+}
+
+const ConnectedAccount = ({ account, onRequestEnable }) => (
+  <div css="margin-top: 12px">
+    {account ? (
+      <LocalIdentityBadge entity={account} />
+    ) : (
+      <Button
+        size="small"
+        mode="secondary"
+        label="Preferences"
+        onClick={onRequestEnable}
+      >
+        <div
+          css={`
+            display: flex;
+            align-items: center;
```

Comment:
Is it not centered properly without this?

Reply:
It is. I've removed it. 

I think it's also not required on the `PreferencesButton` component. Any thoughts?